lib/request-handler.spec.js

```diff
@@ -78,6 +93,46 @@ describe('The request handler', function() {
     })
   })
 
+  describe('the response size limit', function() {
+    beforeEach(function() {
+      camp.route(
+        /^\/testing\/([^/]+)\.(svg|png|gif|jpg|json)$/,
+        handleRequest(standardCacheHeaders, {
+          handler: fakeHandlerWithNetworkIo,
+          fetchLimitBytes: 100,
+        })
+      )
+    })
+
+    it('should not throw an error if the response <= fetchLimitBytes', async function() {
+      nock('https://www.google.com')
+        .get('/foo/bar')
+        .once()
+        .reply(200, 'x'.repeat(100))
+      const res = await fetch(`${baseUrl}/testing/123.json`)
+      expect(res.ok).to.be.true
+      expect(await res.json()).to.deep.equal({
+        name: 'testing',
+        value: '123',
+      })
+      nock.cleanAll()
+    })
+
+    it('should throw an error if the response is > fetchLimitBytes', async function() {
+      nock('https://www.google.com')
+        .get('/foo/bar')
+        .once()
+        .reply(200, 'x'.repeat(101))
+      const res = await fetch(`${baseUrl}/testing/123.json`)
+      expect(res.ok).to.be.true
+      expect(await res.json()).to.deep.equal({
+        name: 'testing',
+        value: 'Maximum response size exceeded',
+      })
+      nock.cleanAll()
```

Comment:
These look good.

This line won't be reached if the `expect`s fail. It's probably safest to put the `nock.cleanAll()` call in an `afterEach` hook.